docs/src/administration/users.md

```diff
@@ -1,65 +1,60 @@
 ---
-title: "User administration"
+title: "Users"
 weight: 1
-sidebarTitle: "Users"
+sidebarTitle: "User administration"
 description: |
-  Every Platform.sh user has a role that controls their access and permission levels. Different roles allow different levels of access to your applications, environments and projects. You can manage how users interact with your project and environments at Platform.sh
+  Every Platform.sh user has a role that controls their access and permission levels.
 ---
 
 {{< description >}}
 
-Any user added to a project or an environment type on Platform.sh will need to [register for an account](https://auth.api.platform.sh/register) before they can contribute. If you need to delete your account at any time you can [transfer ownership](/administration/users.md#transfer-ownership) of your existing projects and then open a [support ticket](/development/troubleshoot.md#deleting-your-platformsh-account) to request your account deletion.  
+If you are an organization or a Project admin wanting to add a user to a project or an environment, the user has to create an account before they can contribute to the project.
 
 ## User roles
 
-Grant user permissions to the entire project:
-* **Project Viewer** - A project viewer can view all environments of the project.
-* **Project Administrator** - A project administrator can change settings, push code and execute actions on all environments of the project.
+| User role    | Description |
+| ------------ |-------------|
+|Project owner| There is only one and they can create multiple Project admins.  |
```

Comment:
I would definitely recommend against standardizing as "Project admin". It may appear that way in the UI, but it should be either a proper noun (Project Admin) or a general one (project admin).

Reply:
Ok, In that case we might want to fix that in the product.